buildtest/buildsystem/base.py

```diff
@@ -274,24 +267,11 @@ def _create_test_folders(self):
         """
         testdir = self._get_testdir()
         create_dir(testdir)
-        for folder in ["run", "log"]:
+        for folder in ["run"]:
             name = "%sdir" % folder
```

Comment:
okay so just to clarify - this "log" folder is removed with preference for putting the buildtest.log directly in the test output folder.

Reply:
Yes there is no log file generated in the testdir and its now just called ``buildtest.log`` in current  directory. Note for now its called ``buildtest.log`` i did this purposely so user can reference one file, it may be useful to remove the file before writing as for now its appending the content.

Reply:
Just FYI. No log file in test directory now

```
(buildtest-framework) ssi29@ag-mxg-hulk090> tree .buildtest
.buildtest
├── python-circle
│   ├── circle_area.py
│   └── run
│       ├── circle_area_04-07-2020-13-19.err
│       ├── circle_area_04-07-2020-13-19.out
│       ├── circle_area_04-07-2020-13-26.err
│       ├── circle_area_04-07-2020-13-26.out
│       ├── circle_area_04-07-2020-13-27.err
│       └── circle_area_04-07-2020-13-27.out
└── slurm
    ├── run
    │   ├── slurm_down_nodes_reason_04-07-2020-13-17.err
    │   ├── slurm_down_nodes_reason_04-07-2020-13-17.out
    │   ├── slurm_down_nodes_reason_04-07-2020-14-59.err
    │   ├── slurm_down_nodes_reason_04-07-2020-14-59.out
    │   ├── slurm_down_nodes_reason_04-07-2020-15-02.err
    │   ├── slurm_down_nodes_reason_04-07-2020-15-02.out
    │   ├── slurm_down_nodes_reason_04-07-2020-15-04.err
    │   ├── slurm_down_nodes_reason_04-07-2020-15-04.out
    │   ├── slurm_not_responding_nodes_04-07-2020-13-17.err
    │   ├── slurm_not_responding_nodes_04-07-2020-13-17.out
    │   ├── slurm_not_responding_nodes_04-07-2020-14-59.err
    │   ├── slurm_not_responding_nodes_04-07-2020-14-59.out
    │   ├── slurm_not_responding_nodes_04-07-2020-15-02.err
    │   ├── slurm_not_responding_nodes_04-07-2020-15-02.out
    │   ├── slurm_not_responding_nodes_04-07-2020-15-04.err
    │   └── slurm_not_responding_nodes_04-07-2020-15-04.out
    ├── slurm_down_nodes_reason.sh
    └── slurm_not_responding_nodes.sh

4 directories, 25 files

```